database/sql/V1.01__events.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS Events;
+
+CREATE TABLE Events (
```

Comment:
@pratibhapaudel Can you please take a look at this schema for Events table and ensure it aligns with the data that will be sent via the frontend?